src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -1217,6 +1217,9 @@ def getChainCreateArgs(firstTask=False, generator=False):
                      'Memory': {'default': None, 'null': True, 'type': float, 'validate': lambda x: x > 0},
                      'Multicore': {'default': 0, 'type': int, 'validate': lambda x: x > 0},
                      'PrepID': {'default': None, 'null': True, 'optional': True, 'type': str},
+                     "RequiresGPU": {"default": "forbidden",
```

Comment:
I hope these few lines  do not overlap with the some of the previous commit. Otherwise this may cause some conflicts.